### ISSUE TITLE
C++ Runtime Error when debuging

### ISSUE THREAD

**Issue by Jack-Works** [Automattic/node-canvas#847]
## Issue or Feature
If I debug my js with a debugger, and I set a breakpoint, then the node-canvas fail and let the whole node fail

## Steps to Reproduce
```js
var Canvas = require('canvas');
var Image = Canvas.Image // Make a breakpoint on this line, I use Visual Studio Code to do this
```

## Your Environment
* Version of node-canvas: 1.6.2
* Environment (e.g. node 4.2.0 on Mac OS X 10.8): Windows 10, Node 6.9

![image](https://cloud.githubusercontent.com/assets/5390719/20456263/49a24f7a-aeae-11e6-82ee-fb1426279705.png)

Thanks

**Comment by zbjornson** [comment]
This might be fixed by #808, if you feel like testing that branch.

**Comment by Jack-Works** [comment]
Cool, thanks!

**Comment by zbjornson** [comment]
Just tested, this is fixed by #808.
